cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/SecretGenerator.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.operator.cluster.operator.assembly;
+
+import io.fabric8.kubernetes.api.model.OwnerReference;
+import io.fabric8.kubernetes.api.model.Secret;
+import io.fabric8.kubernetes.api.model.SecretBuilder;
+import io.strimzi.certs.CertAndKey;
+import io.strimzi.operator.cluster.model.Ca;
+import io.strimzi.operator.cluster.model.ClusterCa;
+import io.strimzi.operator.common.model.Labels;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SecretGenerator {
```

Comment:
This isn't really _generating_ secrets, it's building them. But I guess `SecretBuilder` (although in a different package) was already taken. Can we make this is method of `ModelUtils`?